docs/gateway-api-compatibility.md

```diff
@@ -6,9 +6,9 @@ This document describes which Gateway API resources NGINX Gateway Fabric support
 
 | Resource                            | Core Support Level | Extended Support Level | Implementation-Specific Support Level | API Version |
 |-------------------------------------|--------------------|------------------------|---------------------------------------|-------------|
-| [GatewayClass](#gatewayclass)       | Supported          | Not supported          | Not Supported                         | v1beta1     |
-| [Gateway](#gateway)                 | Supported          | Not supported          | Not Supported                         | v1beta1     |
-| [HTTPRoute](#httproute)             | Supported          | Partially supported    | Not Supported                         | v1beta1     |
+| [GatewayClass](#gatewayclass)       | Supported          | Not supported          | Not Supported                         | v1          |
+| [Gateway](#gateway)                 | Supported          | Not supported          | Not Supported                         | v1          |
+| [HTTPRoute](#httproute)             | Supported          | Partially supported    | Not Supported                         | v1          |
 | [ReferenceGrant](#referencegrant)   | Supported          | N/A                    | Not Supported                         | v1beta1     |
```

Comment:
Is ReferenceGrant not getting promoted to v1?

Reply:
nvm just looked it up and answered my question!

Reply:
I should have called it out - I added it to the description!